### ISSUE TITLE
Derive ExactSizeIterator trait for Positions and others

### ISSUE THREAD

**Issue by farnoy** [alteous/gltf#77]
Right now something like Indices is defined like this: 
```rust
pub enum Indices<'a> {
    U8(Iter<'a, u8>),
    U16(Iter<'a, u16>),
    U32(Iter<'a, u32>),
}
```

And because it exposes Iter, we can use `.len()` easily, but that is not possible for `Positions` - it hides the Iter field.

**Comment by alteous** [comment]
Thanks for reporting. This is clearly an oversight.

Are you using version 0.8.x?

**Comment by farnoy** [comment]
Yes, that's right

**Comment by alteous** [comment]
The code is ready now in the `incoming` branch. I need to update some tests before publishing.

**Comment by farnoy** [comment]
I love the quick response - thank you!

**Comment by alteous** [comment]
[Released in version 0.8.6](https://github.com/alteous/gltf/blob/master/CHANGELOG.md#086---2017-08-26)
